_posts/2018-11-07-arc-gis-server-10-6-upgrade.md

```diff
@@ -25,19 +25,19 @@ We will post the exact date of the upgrade as we finalize our plans. {{ contact
 ### Local Mosaic Datasets
 {: .text-left}
 
-The source data for the Image Services is available for download from [raster.utah.gov](https://raster.utah.gov/). This data allows for the creation of local [mosaic datasets](https://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/what-is-a-mosaic-dataset.htm) for visualization and analysis. The Image Services are a convenience that is no longer going to be offered and the public will have to manage its own mosaic datasets for advanced analysis.
+The source data for the image services on mapserv.utah.gov is available for download from [raster.utah.gov](https://raster.utah.gov/). This data allows for the creation of local [mosaic datasets](https://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/what-is-a-mosaic-dataset.htm) for visualization and analysis. The image services are a convenience that is no longer going to be offered, and the public will have to manage its own mosaic datasets for advanced analysis.
```

Comment:
Regarding: "The image services are a convenience that is no longer going to be offered, and the public will have to manage its own mosaic datasets for advanced analysis."--I wonder whether we can reword this just slightly? Maybe something like 

"The image services themselves, however, will no longer be offered, and other services will need to be used for advanced analysis." 

Or something like that? I don't often love passive voice, but I wonder whether here it would sound just a little more casual/conversational?